drivers/sensor/lm75/lm75.c

```diff
@@ -352,26 +352,24 @@ int lm75_init(const struct device *dev)
 	k_thread_name_set(&data->workq.thread, "lm75_trigger");
 	k_work_init(&data->work, lm75_trigger_work_handler);
 
-	if (cfg->int_gpio.port != NULL) {
-		if (!device_is_ready(cfg->int_gpio.port)) {
-			LOG_ERR("INT GPIO not ready");
-			return -EINVAL;
-		}
-
-		ret = gpio_pin_configure_dt(&cfg->int_gpio, GPIO_INPUT);
-		if (ret < 0) {
-			LOG_ERR("failed to configure INT GPIO (ret %d)", ret);
-			return ret;
-		}
-
-		gpio_init_callback(&data->int_gpio_cb, lm75_int_gpio_callback_handler,
-				   BIT(cfg->int_gpio.pin));
-
-		ret = gpio_add_callback(cfg->int_gpio.port, &data->int_gpio_cb);
-		if (ret < 0) {
-			LOG_ERR("failed to add INT GPIO callback (ret %d)", ret);
-			return ret;
-		}
+	if (!device_is_ready(cfg->int_gpio.port)) {
```

Comment:
This change results in a failure if multiple LM75 devices are present, but only some of them have a trigger GPIO. The NULL check needs to stay.

Reply:
I missed this. Thanks.